session1/examples/generics.ts

```diff
@@ -0,0 +1,33 @@
+//Generic type variables
+function checkPasswordLength<T>(password: T) {
+    console.log(password.length);
+}
+
+//Generic types
+function testArray<T>(item: Array<T>) {
```

Comment:
1. Let's provide a proper function name based on the logic written inside it.
2. Why are using ES5 function syntax here but not the ES6 arrow function?